build.gradle

```diff
@@ -54,14 +54,14 @@ jar {
 }
 
 dependencies {
-    compile gradleApi()
-    compile deps.gson
+    implementation localGroovy()
```

Comment:
Oh, I missed that the plugin you're talking about is `java-gradle-plugin`, I thought it was `java-library-plugin`.  Makes sense!